### ISSUE TITLE
Please enable the addition/inclusion of the tooltip-toggle footnotes from Chrysanthemum Garden when generating epubs from this site.

### ISSUE THREAD

**Issue by TimeLimit67** [dteviot/WebToEpub#1370]
WebToEpub doesn't disappoint when creating epubs from various sites. This includes **Chrysanthemum Garden.** However, the footnotes (_tooltip-toggle footnotes_, not _pop-up footnotes_ like most websites) aren't included in the generated epub.

If possible, I would like a feature added where **these troublesome tooltip-toggle footnotes in every chapter can be included when the epub is generated**. As we all are aware, footntes aid the reader in understanding the story much more, particularly when there are idioms, phrases, terms, etc. that aren't normally understood immediately by readers.

If you know any websites, softwares, extensions, etc. that I can use to achieve this purpose as an alternative, please let me know.

In short, I hope a feature can be added wherein the footnotes in every chapter of a novel from CG will appear on the generated epub.

**Comment by dteviot** [comment]
@TimeLimit67 

To save me time, please give URL of chapter with these, plus where to find on page.

**Comment by yuyu-cloud** [comment]
Hasn't this footnote feature already been added for this site? He also included the option to "remove footnotes" as well by clicking the checkbox. Maybe the novel you're looking at doesn't actually have footnotes? Or you didn't deselect the checkbox when generating the epub?

Also, in case you may have misssed it, but the "tooltip-toggle" footnotes are generated at the very bottom of the xhtml file under each occurring "Translator's Note" per footnote :)

I don't think Chrysanthemum Garden has two types of footnotes, so I'm pretty sure this one is already done xx

**Comment by TimeLimit67** [comment]
Hello!
I see. So that's how it was.

Thanks for the patience. I haven't touched the default settings in WebToEpub so I haven't discovered it. Many thanks for the response.
